.buildkite/pipeline.yml

```
@@ -1,7 +1,7 @@
 # Nodes with values to reuse in the pipeline.
 common_params:
   plugins: &common_plugins
-  - &bash_cache automattic/bash-cache#v1.2.1: ~
+  - &bash_cache automattic/bash-cache#add/pod-actions: ~
```

This branch doesn't exist anymore, does it?

I created a branch on top of this, see below, and CI [failed](https://buildkite.com/wordpress-mobile/wordpresskit-ios/builds/101#604d01ca-1cea-450d-975f-fcfbc5df7173/4-20) with:

> error: pathspec 'add/pod-actions' did not match any file(s) known to git

---

Maybe this?

```suggestion
  - &bash_cache automattic/bash-cache#v1.3.2: ~
```

I can see the "Add pods actions" [commit](https://github.com/Automattic/bash-cache-buildkite-plugin/commit/6adb942a93282d2bf35aa6dc39eb7b613b08de9d) is part of that tag.

---

Great catch, thanks!!